frontend/catalyst/api_extensions/error_mitigation.py

```diff
@@ -30,8 +30,15 @@
 from catalyst.jax_primitives import zne_p
 
 
+def polynomial_extrapolation(degree):
```

Comment:
do we want a default value for the degree?

Reply:
Without knowing the length/size of the scale factors it's hard to know what a reasonable default it. We do end up passing a default once we know that data [here](https://github.com/PennyLaneAI/catalyst/pull/806/files#diff-6cb48a9414798f8b0fc1222b3d3ea0bb73f3981aea277d21f2c69c84b9c13d1aR97-R98).

Reply:
👍 makes sense